src/JsonSchema/Constraints/UndefinedConstraint.php

```diff
@@ -66,7 +66,8 @@ public function validateTypes(&$value, $schema = null, JsonPointer $path, $i = n
         }
 
         // check object
```

Comment:
For some reason, most assoc tests pass, but two of mine fail when strict type checking is used. From what I can see, it looks like properties which contain children simply aren't being parsed, which doesn't seem right...

Currently trying to find the root cause of this - not sure whether the problem is in my code or in master, but starting with the assumption that the problem is mine.